assets/js/index.js

```diff
@@ -18,8 +26,13 @@ searchForm.addEventListener('submit', function (e) {
     //Look in the console to see what value we are getting
     console.log(inputVal)
     //Calls the searchBooks function with a argument of the users input value
-    searchBooks(inputVal)
```

Comment:
add this line back in and this should be good for merge

Reply:
Added!